changelog/new_add_disable_cop_for_entire_file_in_source_code.md

```diff
@@ -0,0 +1 @@
+* [#10856](https://github.com/rubocop/rubocop/pull/10856): Add DisableCopForEntireFileInSourceCode. ([@sambostock][])
```

Comment:
Pending cop name changes, it should contain the department and be surround by code marks in the changelog.

```suggestion
* [#10856](https://github.com/rubocop/rubocop/pull/10856): Add `Style/DisableCopForEntireFileInSourceCode`. ([@sambostock][])
```